driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -192,8 +191,8 @@ private Cluster(
       List<EndPoint> contactPoints,
       Configuration configuration,
       Collection<Host.StateListener> listeners) {
-    System.out.println("===== Using optimized driver!!! =====");
-    logger.info("===== Using optimized driver!!! =====");
+    System.out.println("===== Using Cloud optimized driver!!! =====");
```

Comment:
(not commenting on this specific line) Please add some description to `Enable connectivity with Scylla Cloud clusters` commit.